### ISSUE TITLE
import question, why the model trained by train_ssd_network.py，at last, we can not get anythings after inference?

### ISSUE THREAD

**Issue by zmhik** [balancap/SSD-Tensorflow#349]
this question make me almost crazy, I also read the issues, some body also meet this problem, however, there is no one solve it? can you help me ?

**Comment by chengyangkj** [comment]
> this question make me almost crazy, I also read the issues, some body also meet this problem, however, there is no one solve it? can you help me ?

I had the same problem, the loss of training was around 10, but the model after the training couldn't detect anything.And I can detect things using models trained with voc datasets, but I can't detect things with my own data set.
